BootloaderCorePkg/Stage2/Stage2Hob.c

```diff
@@ -605,6 +605,142 @@ BuildUniversalPayloadHob (
 }
 
 
+/**
+  Build SMM variable related HOBs for Universal Payload
+
+**/
+VOID
+BuildSmmVariableHobs (

```

Comment:
Maybe better to have a PCD to control if we need to build SMM hobs ?
So platform can determine if they want to enable SMM or not.